lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -1145,4 +1145,33 @@ lldb::SBValue Variables::FindVariable(uint64_t variablesReference,
   return variable;
 }
 
+llvm::StringMap<bool> DAP::GetCapabilities() {
+  llvm::StringMap<bool> capabilities;
+
+  // Supported capabilities.
+  capabilities["supportTerminateDebuggee"] = true;
+  capabilities["supportsDataBreakpoints"] = true;
+  capabilities["supportsDelayedStackTraceLoading"] = true;
+  capabilities["supportsEvaluateForHovers"] = true;
+  capabilities["supportsExceptionOptions"] = true;
+  capabilities["supportsLogPoints"] = true;
+  capabilities["supportsProgressReporting"] = true;
+  capabilities["supportsSteppingGranularity"] = true;
+  capabilities["supportsValueFormattingOptions"] = true;
```

Comment:
I'm sure some of these could be associated with a specific request handler. Let me know if anyone feels strongly about moving any of these.

Reply:
I think a lot of those are general enough (like LogPoints that spans all the breakpoint types) that it makes sense to leave them as top level capabilities.